### PR TITLE
Re arrange jobs pages files

### LINKED ISSUE #76: Sitemap.xml is missing /jobs/backend /jobs/frontend routes
**Describe the bug**
If you see the generated sitemap.xml now it doesn't pick sub-paths from `/jobs` such as `/jobs/backend` and `/jobs/frontend`

**Expected behavior**
The sub-routes under jobs are properly picked by sitemap.

### PR DESCRIPTION
**Pull request by tewshi**
Closes #76 

- [x] requires dependency update
- [x] fixes sitemap generation for jobs pages
- [x] creates a layout for jobs (to replace JobsContent component)

### CODE REVIEW

File: layouts/jobs.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@ const css = useCssModule();
 
 <template>
   <PageContainer>
-    <template #title> Jobs </template>
+    <template #title> Jobs</template>
```
Comment (kelsos) [inline layouts/jobs.vue:7]: we need to do something about this editor configuration :P

**Comment by kelsos** [comment]
If we fix the template thing we can merge. @tewshi

**Comment by tewshi** [comment]
> If we fix the template thing we can merge. @tewshi

Reverted @kelsos

**Review by kelsos — APPROVED** [review]
lgtm @tewshi thanks

**Pull request merged**